db/migrate/20210421121431_add_case_insensitive_btree_index_to_tags.rb

```diff
@@ -1,8 +1,25 @@
 class AddCaseInsensitiveBtreeIndexToTags < ActiveRecord::Migration[5.2]
   disable_ddl_transaction!
 
+  class CorruptionError < StandardError
```

Comment:
The class defined here is referenced by another migration. I think it would be best to move the definition to a neutral file.

Reply:
I'll see how to refactor that.

Reply:
Refactored that.